python/kwiver/sprokit/tests/sprokit/pipeline/CMakeLists.txt

```diff
@@ -21,14 +21,20 @@ set(__simple_testable_modnames
 
 # KWIVER_ENABLE_TESTS is always true for now (because we are in the test
 # subdir), but the python tests should be moved to the python source dir.
-if (KWIVER_ENABLE_TESTS)
-  include(kwiver-test-setup)
-  # Register individual tests with ctest
+if (KWIVER_ENABLE_TESTS AND PYTEST_RUNNER)
+
+# Register individual tests with ctest
```

Comment:
Again, really minor but the space before the comment should stay so intending is correct.